### PR TITLE
create simple product without sku

### PR DESCRIPTION
**Pull request by szczecha**
I want to merge this change because it fixes test `SALEOR_2806 should create simple product without sku` 

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://automation-dashboard.staging.saleor.cloud/graphql/
MARKETPLACE_URL=https://marketplace-gray.vercel.app/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-7715 Fix test should create simple product without sku](https://app.clickup.com/t/2549495/SALEOR-7715)

**Review by eCzerniak — CHANGES_REQUESTED** [review]
Need to check after merging PR with fix from dev side.

**Comment by szczecha** [comment]
> Need to check after merging PR with fix from dev side.

@eCzerniak fix for publishing products was merged and I rebased branch with main

File: cypress/e2e/products/productsWithoutSku/createProductWithoutSku.js [hunk 1]
```diff
@@ -193,7 +193,7 @@ describe("Creating variants", () => {
   );
 
   it(
-    "should create simple product without sku",
+    "should create simple product without sku SALEOR_2806",
     { tags: ["@products", "@allEnv"] },
     () => {
       const name = `${startsWith}${faker.datatype.number()}`;
```
Comment (eCzerniak) [inline cypress/e2e/products/productsWithoutSku/createProductWithoutSku.js:199]:
```suggestion
      const name = `${startsWith}${faker.datatype.number()}`;
      
```

File: cypress/e2e/products/productsWithoutSku/createProductWithoutSku.js [hunk 2]
```diff
@@ -227,15 +233,13 @@ describe("Creating variants", () => {
...
         .click()
         .confirmationMessageShouldDisappear()
-        .wait("@VariantCreate")
+        .wait("@ProductDetails")
         .then(({ response }) => {
-          const variants = [
-            response.body.data.productVariantCreate.productVariant,
-          ];
+          const variants = [response.body.data.product.variants[0]];
           createWaitingForCaptureOrder({
```
Comment (eCzerniak) [inline cypress/e2e/products/productsWithoutSku/createProductWithoutSku.js:243]:
```suggestion
          createWaitingForCaptureOrder({
             channelSlug: defaultChannel.slug,
             email: "example@example.com",
             variantsList: variants,
             shippingMethodName: shippingMethod.name,
             address,
            })
              .its("order.id")
              .should("be.ok");;
```

**Review by eCzerniak — APPROVED** [review]
Please consider my comments. Besides that it is ok.

**Review by karola312 — APPROVED** [review]

**Pull request merged**